mmdet3d/datasets/sensaturban_dataset.py

```diff
@@ -0,0 +1,124 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import os.path as osp
+from typing import Callable, List, Optional, Union
+
+from mmdet3d.registry import DATASETS
+from .seg3d_dataset import Seg3DDataset
+
+
+@DATASETS.register_module()
+class SensatUrbanDataset(Seg3DDataset):
+    r"""SensatUrban Dataset.
+
+    This class serves as the API for experiments on the SensatUrban Dataset
+    Please refer to <https://urban3dchallenge.github.io>`_
```

Comment:
refer to \`its official website \<https://urban3dchallenge.github.io>`_ for more details.